ui/js/models/log_slice.js

```diff
@@ -67,82 +67,6 @@ treeherder.factory('ThLogSliceModel', [
         }
     };
 
-    ThLogSliceModel.prototype.load_more = function (bounds, element) {
```

Comment:
yeah.  this again was part of the "code cleanup" commit.  Something I fixed while I was here.  Perhaps we should do "code cleanups" in separate PRs to make this simpler?